pages/about.tsx

```diff
@@ -1,11 +1,42 @@
 import { VStack } from '@chakra-ui/react'
+import { InferGetStaticPropsType } from 'next'
+import { RichText } from '@graphcms/rich-text-react-renderer'
 
 import { Heading } from 'components/Base/Heading'
-import { HighlightLink } from 'components/Base/HighlightLink'
+import { ImageWithCaptions } from 'components/ImageWithCaptions'
+import { getAboutMePage } from 'graphql/queries/getAboutMePage'
 import { Paragraph } from 'components/Base/Paragraph'
-import { PlaceImage } from 'components/PlaceImage'
+import { HighlightLink } from 'components/Base/HighlightLink'
+
+export const getStaticProps = async () => {
+  try {
+    const { sections } = await getAboutMePage()
+
+    return {
+      props: {
+        sections
+      }
+    }
+  } catch (error) {
+    // eslint-disable-next-line no-console
+    console.log(error)
```

Comment:
Those `console.log` with ESLInt disable comments are not looking good, so I also have a task in the project board to integrate with a proper monitoring tool such as Sentry